scan_test.go

```diff
@@ -296,3 +296,231 @@ func BenchmarkZoneParser(b *testing.B) {
+				"r _H _BzOc6S_Lk0Ky43" +
+				"j1TzV__9367tbX56_6B3" +
+				"__q6_v8_4_0_t_2q_nJ2" +
+				"gV3j9_tkOrx_H__ 4530" +
+				"MtN8BkQwZJNxz_7  D32" +
+				" _v7m 3V0Uok 3GENERA" +
+				"TEA53\t6e48562e0xF\tE\t" +
+				"   -0xcFee-EZ_z \t\t-0" +
+				"xb1AF1Fb71D2ACeaB3FE" +
+				"ce2s   3Al 6e-5 \t \t\t" +
+				"N_W6J3PBS_W__CyJu__k" +
+				"6F_jY0INI_LC277x14b_" +
+				"1b___Y8f_K_3y_0055ya" +
+				"P_LKu_72g_T_32iBk1Zm" +
+				"_o \x01 xOu __BzOc6S_Lk" +
+				"0Ky43j1TzV__9367tbX5" +
+				"6_6B3__q6_v8_4_0_t_2" +
+				"q_nJ2gV3j9_tkOrx_H__" +
+				"-0xb1AF1Fb71D2ACeaB3" +
+				"FEce2s  xlT3KTf_dr 2" +
+				"7 \t \t\tN_W6J3PBS_W__C" +
+				"yJu__k6F_jY0INI_LC27" +
+				"7x14b_1b___Y8f_K_3y_" +
+				"0055yaP_LKu_72g_T_32" +
+				"iBk1Zm_o  sQ_9_3\t \tm" +
+				" aa_XO4_5-03575240.-" +
+				"045353LOT_o_BzOc6S_L" +
+				"k0Ky43j1TzV__9367tbX" +
+				"56_6B3__q6_v8_4_0_t_" +
+				"2q_nJ2gV3j9_tkOrx_H_" +
+				"_  m aa_XO4_5  \t\t\t\t_" +
+				"_gWZo90gm1sP9_BdL_5_" +
+				"kY___U0S_J1iDk_o_h88" +
+				"4o5_jn_085HZCnBj1b73" +
+				"\t\t  m\t  \t\t\tsMPr0_r4L" +
+				"rd362p_O_GN0_2_JWohR" +
+				"WFEj7_DJi3oRq_6bGEln" +
+				"KBl9tad_5Js5Ss8_PDrL" +
+				"2nHnE90BqH24_QiMnY9 " +
+				"-2371737156m 7e mP_Q" +
+				"q_3sJ_1_84X_545iW_3K" +
+				"4p8J8_v9_LT3_6_0l_3D" +
+				"4VT3xq6N_3K___U_xX2 " +
+				"Fv1_j yGgp_1E_Fb_t_3" +
+				"__G02 _mgc_3G1X_7i_g" +
+				"_7G4M25_lh__VR0____p" +
+				"_R_x_AwC5CEfE_f_zjRt" +
+				"7_p6c__1_fh_NJm6T7qA" +
+				"_  ,1 0A5__S_xf7___0" +
+				"XXp__KFjRV__JU3o,0xb" +
+				"1AF1Fb71D2ACeaB3FEce" +
+				"2s\tsg xlT3KTf_dr-6 \t" +
+				" \t\tN_W6J3PBS_W__CyJu" +
+				"__k6F_jY0INI_LC277x1" +
+				"4b_1b___Y8f_K_3y_005" +
+				"5yaP_LKu_72g_T_32iBk" +
+				"1Zm_osQ_9_3\t a m aa_" +
+				"XO4_5\t g6bp_KI_1_YWc" +
+				"_K8c2b8_2_5k_BzOc6S_" +
+				"Lk0Ky43j1TzV__9367tb" +
+				"X56_6B3__q6_v8_4_0_t" +
+				"_2q_nJ2gV3j9_tkOrx_H" +
+				"__  m aa_XO4_5  \t\t\t\t" +
+				"d\t\t  m sMPr0_r4Lrd36" +
+				"2p_O_GN0_2_JWohRWFEj" +
+				"7_DJi3oRq_6bGElnKBl9" +
+				"tad_5Js5Ss8_PDrL2nHn" +
+				"E90BqH24_QiMnY9 -783" +
+				".03260xEd55d0Fe3Fe04" +
+				"32 m 7e\\-16855948535" +
+				"74011675064937138012" +
+				"228 Fv1_j e x3es  ,Q" +
+				"5519  _h4hQ_BzOc6S_L" +
+				"d780k_z24_wF_gHD_m37" +
+				"g_YWZ12W\"\"c_K8c2b _B" +
+				"zOc6S_Lk0Ky43j1TzV__" +
+				"9367tbX56_6B3__q6_v8" +
+				"_4_0_t_2q_nJ2gV3j9_t" +
+				"kOrx_H__\t\ta\t \t\t0A3_b" +
+				"8_0I_GTKL_LC_pQ2MN_G" +
+				"Y9YKp6H_cL_0V Rsg6bp" +
+				"_KI_1_YWc_K8c2b _BzO" +
+				"c6S_Lk0Ky43j1TzV__93" +
+				"67tbX56_6B3__q6_v8_4" +
+				"_0_t_2q_nJ2gV3j9_tkO" +
+				"rx_H__ a\tJ  \a\xfde s 60" +
+				"itr s t td  M__6VU_S" +
+				"B45BVM2i69 W__z0AEfP" +
+				"_g6me \"\"2Z_qY05APO_0" +
+				"SgAvjFYC_0__0riU_9K9" +
+				"_C__6af__wj_UbSYy_ge" +
+				"29S_s_Qe259q_kGoddr\t" +
+				" \t\t^  \t\t \t ;;;;;;;;;" +
+				";;;;;;;;;;",
+			`dns: comment length insufficient for parsing: "comment length insufficient for parsing" at line: 1:2030`,
```

Comment:
are these 2 inputs actually testing the same thing? As they are quite long, can we just use 1?